Show EA 3 deprecation message

charangit opened the pull request:
Case ZC-3956: EA 3 deprecation message is shown in the security advisor
with details about the date of deprecation and the removal of its support
from cPanel & WHM v78.

357r4bd reviewed (commented):
Sent comment privately.

357r4bd commented:
Looks good to me.

Commit:
Show EA 3 deprecation message

Case ZC-3956: EA 3 deprecation message is shown in the security
advisor with details about the date of deprecation and the removal of its support
from cPanel & WHM v78.
